src/razor/src/semantic/semanticTokensRangeHandler.ts

```diff
@@ -18,8 +17,8 @@ export class SemanticTokensRangeHandler {
         any
     > = new RequestType(SemanticTokensRangeHandler.getSemanticTokensRangeEndpoint);
     private emptySemanticTokensResponse: ProvideSemanticTokensResponse = new ProvideSemanticTokensResponse(
-        new SemanticTokensResponse(new Array<number>(), ''),
-        null
+        new Array<Array<number>>(),
+        1
```

Comment:
Or maybe -1 because this is really not meant to be used.

```suggestion
        -1
```